cmd/control/control.go

```diff
@@ -640,17 +618,12 @@ var (
 
 func newSendOutboundMessageCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "send-outbound-message [options]",
-		Short: "Sends an outbound message to an attached peer.",
-		RunE:  sendOutboundMessageFunc,
-		Args:  cobra.ExactArgs(0),
+		Use:       "send-outbound-message [options]",
```

Comment:
I think you're missing a `node-name` here

Reply:
you're right, fixed.